js/src/forum/pollyfills.js

```diff
@@ -0,0 +1,17 @@
+if (!String.prototype.startsWith) {
+  Object.defineProperty(String.prototype, 'startsWith', {
+    value: function(search, pos) {
+      pos = !pos || pos < 0 ? 0 : +pos;
+      return this.substring(pos, pos + search.length) === search;
+    }
+  });
+}
```

Comment:
Where are these taken from? Any other licenses we need to mention?

Reply:
These are taken from the MDN polyfils @ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Polyfill & https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#Polyfill.

However, it looks like they are taken from https://github.com/mathiasbynens/String.prototype.startsWith @ https://github.com/mathiasbynens/String.prototype.endsWith respectively, so perhaps we want to use the NPM packages instead to avoid licensing issues.